code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -790,3 +790,148 @@ Basically, we fill the time between now and 2s from now with hands based off the
 		return
 
 	hearing_args[HEARING_RAW_MESSAGE] = "<span class='[randomSpan]'>[hearing_args[HEARING_RAW_MESSAGE]]</span>"
+
+/datum/reagent/inverse/sal_acid
+	name = "Benzoic Acid"
+	description = "Robust fertilizer that provides a decent range of benefits for plant life."
+	taste_description = "flowers"
+	reagent_state = LIQUID
+	color = "#e6c843"
+	ph = 3.4
+	tox_damage = 0
+
+/datum/reagent/inverse/sal_acid/on_hydroponics_apply(obj/item/seeds/myseed, datum/reagents/chems, obj/machinery/hydroponics/mytray)
+	. = ..()
+	if(myseed && chems.has_reagent(src.type, 1))
+		myseed.adjust_potency(round(chems.get_reagent_amount(src.type) * 0.25))
+		myseed.adjust_yield(round(chems.get_reagent_amount(src.type) * 0.2))
+		myseed.adjust_production(-round(chems.get_reagent_amount(src.type) * 0.2))
+		mytray.adjust_plant_health(round(chems.get_reagent_amount(src.type) * 0.5))
```

Comment:
I'm not sure if this code is old or you're upstreaming this from an older codebase but this is the outdated form of `on_hydroponics_apply` and won't work. 

This is how the proc is structured now. 

![image](https://github.com/tgstation/tgstation/assets/51863163/84bf1959-1eaf-4385-aaf2-2d474b73d898)